frontend/src/concepts/pipelines/content/createRun/contentSections/ExperimentSection.tsx

```diff
@@ -1,57 +1,44 @@
 import * as React from 'react';
-import { Button, FormGroup, FormSection, Stack, StackItem } from '@patternfly/react-core';
+import { FormGroup, FormSection, Stack, StackItem } from '@patternfly/react-core';
 import { PlusCircleIcon } from '@patternfly/react-icons';
 import { ExperimentKFv2 } from '~/concepts/pipelines/kfTypes';
-import ManageExperimentModal from '~/concepts/pipelines/content/experiment/ManageExperimentModal';
-import { usePipelinesAPI } from '~/concepts/pipelines/context';
 import {
   CreateRunPageSections,
   runPageSectionTitles,
 } from '~/concepts/pipelines/content/createRun/const';
 import ExperimentSelector from '~/concepts/pipelines/content/experiment/ExperimentSelector';
+import CreateExperimentButton from '~/concepts/pipelines/content/experiment/CreateExperimentButton';
 
 type ExperimentSectionProps = {
   value: ExperimentKFv2 | null;
   onChange: (experiment: ExperimentKFv2) => void;
 };
 
-const ExperimentSection: React.FC<ExperimentSectionProps> = ({ value, onChange }) => {
-  const { refreshAllAPI } = usePipelinesAPI();
-  const [openCreate, setOpenCreate] = React.useState(false);
-  return (
-    <>
-      <FormSection
-        id={CreateRunPageSections.EXPERIMENT}
-        title={runPageSectionTitles[CreateRunPageSections.EXPERIMENT]}
-      >
-        {/* `minWidth` a temp fix for PF issue https://github.com/patternfly/patternfly/issues/6062
+const ExperimentSection: React.FC<ExperimentSectionProps> = ({ value, onChange }) => (
+  <FormSection
+    id={CreateRunPageSections.EXPERIMENT}
+    title={runPageSectionTitles[CreateRunPageSections.EXPERIMENT]}
+  >
+    {/* `minWidth` a temp fix for PF issue https://github.com/patternfly/patternfly/issues/6062
```

Comment:
Are we sure this is still needed? What happens when we remove this inline style?